frontend/src/lang/en.ts

```diff
@@ -0,0 +1,14 @@
+export const en = {
+  translation: {
+    "Patient header": "COVID-19 Testing Portal",
+    "Register for your test": "Register for your test",
+    "Registration complete": "Registration complete",
+    terms: {
```

Comment:
We can define these objects pretty flexibly to avoid collisions and name things intuitively